guidance/content/regulations/recent-changes.html

```diff
@@ -0,0 +1,11 @@
+{% comment %}
+    Prototype for recent changes display
+{% endcomment %}
+
+<div class="recent-changes">
+    <h1>Recent Changes: Amended 11/06/2020</h1>
+    <a class="change-link" target="_blank" href="https://www.ecfr.gov/cgi-bin/text-idx?SID=fbe41d98a4ab30061e60b186e69f1b20&mc=true&node=se42.4.433_1400&rgn=div8">
```

Comment:
I'll work on getting the content up in dev but here's the link for this section: `/433-400/2020-28567#433-400`

```suggestion
    <a class="change-link" target="_blank" href="/433-400/2020-28567#433-400">
```

Reply:
Thanks!